cookbooks/aws-parallelcluster-install/recipes/user.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: user
+#
+# Copyright 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setup byos group
+group node['cluster']['byos']['group'] do
+  comment 'byos group'
+  gid node['cluster']['byos']['group_id']
+  system true
+end
+
+# Setup byos user
+user node['cluster']['byos']['user'] do
```

Comment:
can we configure the byos user as a sudoer by default until we make it configurable?

Reply:
Are there commands we want to restrict it by? I think nopasswd: all would basically open up the world (which we could clamp down later) -- but I want to verify if that's what we want to do or if there's restrictions we should put in place.